test/system/chatrooms_test.rb

```
@@ -0,0 +1,51 @@
+    page.evaluate_script(<<~JS)
+      (() => {
+        const el = document.querySelector(
+          ".event-#{chatroom.id} [data-chatroom-subscription-target='latestMessages']"
+        );
+        if (!el) return false;
+        const overflowing = el.scrollHeight > el.clientHeight;
+        const atBottom = Math.abs(el.scrollHeight - el.clientHeight - el.scrollTop) < 5;
+        return overflowing && atBottom;
+      })()
+    JS
+  end
```

RuboCop (1.88.0)

[convention] Method has too many lines. [11/10]

(Metrics/MethodLength)